Author: Jasper Grimm <jasper.grimm@york.ac.uk>
Description: avoid need for submodule, and rely on morse_cmake via
             CMAKE_MODULE_PATH
diff -ruN pastix-v6.2.2.orig/CMakeLists.txt pastix-v6.2.2/CMakeLists.txt
--- pastix-v6.2.2.orig/CMakeLists.txt	2022-09-08 13:25:28.000000000 +0100
+++ pastix-v6.2.2/CMakeLists.txt	2022-12-22 14:13:02.000000000 +0000
@@ -61,9 +61,9 @@
     endif()
 endif()
 
-if((NOT EXISTS "${PROJECT_SOURCE_DIR}/spm/CMakeLists.txt") OR (NOT EXISTS "${PROJECT_SOURCE_DIR}/cmake_modules/morse_cmake/modules/MorseInit.cmake"))
-    message(FATAL_ERROR "The submodules were not downloaded! GIT_SUBMODULE was turned off or failed. Please update submodules and try again.")
-endif()
+# if((NOT EXISTS "${PROJECT_SOURCE_DIR}/spm/CMakeLists.txt") OR (NOT EXISTS "${PROJECT_SOURCE_DIR}/cmake_modules/morse_cmake/modules/MorseInit.cmake"))
+#     message(FATAL_ERROR "The submodules were not downloaded! GIT_SUBMODULE was turned off or failed. Please update submodules and try again.")
+# endif()
 
 # Add extra cmake module path and initialize morse cmake modules
 # --------------------------------------------------------------
@@ -75,7 +75,7 @@
   list(APPEND CMAKE_MODULE_PATH ${MORSE_CMAKE_MODULE_PATH})
   include(MorseInit)
 else()
-  message(FATAL_ERROR "Submodule cmake_morse not initialized - run `git submodule update --init`")
+  include(MorseInit)
 endif()
 
 # Define precision supported by PaStiX
